securedrop/requirements/update_python_dependencies

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Usage: ./update-python-dependencies
```

Comment:
I just manually triggered a rebuild of the "latest" docs from the Read the Docs administrative interface, and https://docs.securedrop.org/en/latest is back. Not sure what went wrong there. Skimmed the build logs but don't see anything unusual. We should keep an eye on it in case it happens again.